pkgs/applications/video/lightworks/default.nix

```
@@ -11,22 +11,25 @@ let
     libjpeg_original
     glib
     pango
+    libGL
     libGLU
     nvidia_cg_toolkit
     zlib
     openssl
-    portaudio
+    libuuid
+    alsaLib
+    udev
   ];
 
   lightworks = stdenv.mkDerivation rec {
-    version = "14.0.0";
+    version = "2021.2";
     pname = "lightworks";
 
     src =
       if stdenv.hostPlatform.system == "x86_64-linux" then
         fetchurl {
-          url = "http://downloads.lwks.com/v14/lwks-14.0.0-amd64.deb";
-          sha256 = "66eb9f9678d979db76199f1c99a71df0ddc017bb47dfda976b508849ab305033";
+          url = "https://cdn.lwks.com/releases/2021.2/lightworks_2021.2_r128258.deb";
```

```suggestion
          url = "https://cdn.lwks.com/releases/${version}/lightworks_${version}_r128258.deb";
```

the rXXXX part could be replaced with an update script.

---

I am sorry, I don't fully understand. Could you please elaborate?